src/components/error-config-not-supported/error-config-not-supported.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { shallow, Wrapper } from '@vue/test-utils';
+
+import { renderComponent } from '../../../tests/helpers/render';
+import { ERROR_TEMPLATE_NAME } from '../component-names';
+import { MErrorConfigNotSupported } from './error-config-not-supported';
+
+let wrapper: Wrapper<MErrorConfigNotSupported>;
+
+const getStubs: any = () => {
+    return {
+        [ERROR_TEMPLATE_NAME]: '<div><slot /></div>'
+    };
+};
+
+describe(`Page not found - test`, () => {
```

Comment:
Config not supported - Test au lieu de Page not found - Test?

Reply:
done